### DISCUSSION TITLE
Rencana Layout pejuangkode.com

### DISCUSSION THREAD

**Discussion by ekaone** [pejuang-kode/pejuangkode.com#3] [General]
Untuk halaman depan (Landing Page), dari [Tailwind UI Components](https://tailwindui.com/components) ini, Mas @zainfathoni rencananya mau pakai yang mana..?.

Thanks 👌

**Comment by zainfathoni** [comment]
Saya juga belum putuskan sih, akhir pekan ini _insya Allah_ akan saya putuskan sekaligus pasang layout-nya di repository Mas @ekaone. 😉

**Reply by ekaone** [reply]
ok siap 😀
